app/src/main/java/io/neurolab/activities/MeditationActivity.java

```diff
@@ -44,6 +45,14 @@ protected void onCreate(Bundle savedInstanceState) {
         initializePlaybackController();
     }
 
+    @Override
+    public void onConfigurationChanged(Configuration newConfig) {
```

Comment:
Please remove this function. The scrollview is not working because someone set `android:configChanges="orientation|screenSize"` for MeditationActivity. We need to improve the codebase by a large margin, Ill start opening issues soon. You can remove this function and remove configChanges attritbute for this activity and it should work just with the landscape layout